cpu/stm32f0/include/vendor/stm32f030x8.h

```diff
@@ -4789,6 +4789,9 @@ typedef struct
 #define USART_CR1_EOBIE_Pos           (27U)
 #define USART_CR1_EOBIE_Msk           (0x1U << USART_CR1_EOBIE_Pos)            /*!< 0x08000000 */
 #define USART_CR1_EOBIE               USART_CR1_EOBIE_Msk                      /*!< End of Block interrupt enable */
+#define USART_CR1_M1_Pos              (28U)
```

Comment:
7 bit mode is not supported by stm32f030x4/6/8, see reference manual DM00091010, table 85, page 597.

So I would not add the `USART_CR1_M1` definitions here.